packages/manager/modules/pci/src/projects/project/storages/objects/object/addUser/addUser.routing.js

```diff
@@ -1,43 +1,52 @@
 export default /* @ngInject */ ($stateProvider) => {
-  $stateProvider.state('pci.projects.project.storages.objects.addUser', {
-    url: '/addUser?containerId&isHighPerfStorage',
-    views: {
-      modal: {
-        component: 'pciProjectStorageContainersContainerAddUser',
+  $stateProvider.state(
+    'pci.projects.project.storages.objects.objects.addUser',
+    {
+      url: '/addUser?containerId&isHighPerfStorage',
+      views: {
+        modal: {
+          component: 'pciProjectStorageContainersContainerAddUser',
+        },
       },
-    },
-    redirectTo: (transition) =>
-      transition
-        .injector()
-        .getAsync('availableUsers')
-        .then((availableUsers) =>
-          availableUsers.length === 0
-            ? { state: 'pci.projects.project.storages.objects.emptyUser' }
-            : false,
-        ),
-    layout: 'modal',
-    resolve: {
-      isHighPerfStorage: /* @ngInject */ ($transition$) =>
-        $transition$.params().isHighPerfStorage === 'true',
-      containerId: /* @ngInject */ ($transition$) =>
-        $transition$.params().containerId,
-      availableUsers: /* @ngInject */ ($http, projectId) =>
-        $http.get(`/cloud/project/${projectId}/user`).then(({ data }) => data),
-      container: /* @ngInject */ (
-        PciProjectStorageContainersService,
-        projectId,
-        containerId,
-        isHighPerfStorage,
-      ) =>
-        PciProjectStorageContainersService.getContainer(
+      redirectTo: (transition) =>
+        transition
+          .injector()
+          .getAsync('availableUsers')
+          .then((availableUsers) =>
+            availableUsers.length === 0
+              ? {
+                  state:
+                    'pci.projects.project.storages.objects.objects.emptyUser',
+                }
+              : false,
+          ),
+      layout: 'modal',
+      resolve: {
+        isHighPerfStorage: /* @ngInject */ ($transition$) =>
+          $transition$.params().isHighPerfStorage === 'true',
+        containerId: /* @ngInject */ ($transition$) =>
+          $transition$.params().containerId,
+        availableUsers: /* @ngInject */ ($http, projectId) =>
+          $http
+            .get(`/cloud/project/${projectId}/user`)
+            .then(({ data }) => data),
+        container: /* @ngInject */ (
```

Comment:
this is already defined in 
packages/manager/modules/pci/src/projects/project/storages/object-storage/objects/object/addUser/addUser.routing.js
can you reuse it plz